ci/tasks/test-e2e.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
We are re-using this `sync-to-docker-vm` code in many places. Its probably a candidate to extract into a helper script and bring in from `concourse-shared` repo cc @sandipndev 

Would leave it like this for this PR as that is an orthogonal task.